components/review-modal/review-modal.types.ts

```diff
@@ -1,12 +1,13 @@
-import { MainModalProps } from '../main-modal'
+import { Network } from '../network-box/network-box.types'
 
 export type ReviewModalProps = {
   closeModal: () => void
-} & Pick<
-  MainModalProps,
-  | 'handleBridgeToEthereum'
-  | 'amount'
-  | 'ethereumAddress'
-  | 'tariWalletAddress'
-  | 'isBridging'
->
+  handleBridgeToEthereum: () => void
+  handleBridgeToTari: () => void
+  isBridging: boolean
```

Comment:
```suggestion
  & Pick<
  MainModalProps,
  | 'handleBridgeToEthereum'
  | 'amount'
  | 'ethereumAddress'
  | 'tariWalletAddress'
  | 'isBridging'
>
```

Since we are drilling props down to the child component, it's safer to inherit shared props from the parent.